File: Prefix Sum/1480. Running Sum of 1d Array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var runningSum = function(nums) {
    var arr=new Array();
    arr[0]=nums[0];
    for(var i=1;i<nums.length;i++)
        {
            arr[i]=arr[i-1]+nums[i];
        }
    return arr;
    
};

File: Amazon/Easy/1480. Running Sum of 1d Array.cpp
class Solution {
public:
    vector<int> runningSum(vector<int>& nums) {
        vector<int>v1(nums.size());
        v1[0]=nums[0];
        for(int i=1;i<nums.size();i++)
        {
           v1[i]=nums[i]+ v1[i-1];
        }
        return v1;
    }
};
